server/videos-full-stack.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE videos 
+    (
+        id SERIAL PRIMARY KEY,
+        title VARCHAR(50) NOT NULL,
```

Comment:
You only allow titles of up to 50 characters in your database schema - what would happen if someone tried to post a new video with a longer title? What would you like/expect to happen?